packages/builder/src/components/backend/DataTable/modals/ExportModal.svelte

```diff
@@ -15,8 +22,65 @@
   ]
 
   export let view
+  export let filters
+  export let sorting
+  export let selectedRows = []
 
   let exportFormat = FORMATS[0].key
+  let filterLookup
+  let exportOpDisplay = []
+
+  $: luceneFilter = LuceneUtils.buildLuceneQuery(filters)
+
+  $: if (filters) {
+    exportOpDisplay = filterDisplay()
+  }
+
+  $: if (sorting) {
+    exportOpDisplay.push({
```

Comment:
Running an array.push in a reactive statement like this could be bug prone, because if the reactive statement runs more than once then it will end up pushing additional duplicates onto the array. I would either passing `sorting` as a param to the `filterDisplay` function which creates `exportOpDisplay` and optionally append it, or use a different reactive variable which spreads `exportOpDisplay` and optionally appends another object.

Either way, you just want to ensure that it can't create duplicates from further invocations of the reactive statement.